web/src/global/GlobalAlerts.tsx

```diff
@@ -2,6 +2,7 @@ import { parseISO } from 'date-fns'
 import differenceInDays from 'date-fns/differenceInDays'
 import * as React from 'react'
 import { Subscription } from 'rxjs'
+import { major, minor, parse } from 'semver'
```

Comment:
I would recommend using a `*` import here to make it clear what these functions are parsing at the call site
```suggestion
import * as semver from 'semver'
```